package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-core",
-  "version": "2.6.0-beta",
+  "version": "2.6.0-beta.2",
```

Comment:
silly question - is it normal/typical to go to .2 instead of .1? I'm not sure how versioning works

Reply:
I'm not sure neither, but I was told to go from x.x.x-beta to x.x.x-beta.2
I have already merged the beta.2 version for search-headless before, so I assume this is ok?

Reply:
I had the same question but I'm not sure either. Maybe Yen knows? If you've done it before, perhaps it's ok.

Reply:
Yen did told me that there is no strict rule on the revision numbers on beta. She personally view the original as the first version so next up should be .2